examples/react/forms.html

```diff
@@ -2,16 +2,26 @@
 <html>
   <head>
     <link rel="stylesheet" type="text/css" href="../assets/mui/css/mui.css"/>
-    <script src="../assets/vendor/react/0.12.2/react.js"></script>
     <script src="../assets/mui/react/mui-react.js"></script>
     
-    <!-- for example only -->
-    <script src="../assets/vendor/jquery-1.11.2.min.js"></script>
-    <script src="../assets/vendor/react/0.12.2/JSXTransformer.js"></script>
+    <!-- for example only -->   
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.2/react.js"></script>
```

Comment:
With this change I don't think we don't need the examples/assets/vendor directory anymore.